test/pages.js

```diff
@@ -2112,7 +2114,7 @@ describe('Pages', function() {
           slug: '/level-1-page-2',
           path: `${home.aposDocId}/${level1Page2.aposDocId}`,
           level: 1,
-          rank: 4,
+          rank: 3,
```

Comment:
I went through this and determined that these new relative ranks are correct. It's just that the new code leaves fewer gaps between ranks.